docker/README.md

```diff
@@ -120,199 +78,414 @@ mv license.key "${LOCAL_LICENSE_FILE_DIR}/license"
-- Add the following CAS Service configuration variables
-  - `CAS_BASE_URL`
-  - `CAS_BIND_ADDRESS`
-  - `CAS_BIND_PORT`
-  - `CAS_DATABASE_NAME`
-  - `CAS_DEBUG`
-  - `CAS_DEFAULT_USER_ROLE`
+Next, you will need to place a **reverse proxy or SSL endpoint** in front of your FiftyOne 
+Enterprise services. This can be a tool like:
 
-> **NOTE**: When multiple deployments use the same database instance,
-> set `CAS_DATABASE_NAME` to a unique value for each deployment.
+- **Nginx**
+- **HAProxy**
+- **Cloud Load Balancer** (e.g., AWS ALB, GCP Load Balancer)
 
-Please review these changes in the
-[legacy-auth/env.template](legacy-auth/env.template)
-and in the appropriate `legacy-auth/compose*` files.
+These proxies will:
 
-To upgrade from versions prior to FiftyOne Enterprise v1.6
+- Route traffic to the correct services (`teams-app`, `teams-api`, `teams-cas`)
+- Terminate HTTPS/SSL (if using TLS)
+- Optionally apply authentication headers, logging, or load balancing
+
+### 🧭 Routing Overview (Path-Based Proxy)
+
+| Path              | Proxied To       | Description                          |
+|-------------------|------------------|--------------------------------------|
+| `/`               | `teams-app`      | Main web UI                          |
+| `/cas`            | `teams-cas`      | Central Authentication Service (CAS) |
+| `/graphql/v1`     | `teams-api`      | GraphQL API endpoint                 |
+| `/file`           | `teams-api`      | File import handling                 |
+| `/_pymongo`       | `teams-api`      | MongoDB requests via SDK             |
+| `/health`         | `teams-api`      | Health check endpoint                |
+
+### 📁 Nginx Configuration Options
+
+Voxel51 provides example Nginx configs for two routing strategies:
+
+#### 🔹 1. **Path-Based Routing**
+All services are routed based on URL path:
+
+📄 Full configuration here: [`example-nginx-path-routing.conf`](./example-nginx-path-routing.conf)
+
+#### 🔹 2. **Hostname-Based Routing**
+teams-app and teams-api are routed using different subdomain or hostname:
+
+- `fiftyone.your-company.com` → App
+- `fiftyone-api.your-company.com` → API
+
+📄 Full configuration here:
+- [`example-nginx-site.conf`](./example-nginx-site.conf) (App + CAS)
+- [`example-nginx-api.conf`](./example-nginx-api.conf) (API)
+
+
+### 📌 Notes
+
+- FiftyOne Enterprise supports routing traffic through proxy servers. Please refer to the 
+[proxy configuration documentation](./docs/configuring-proxies.md) for information on how to configure proxies.
+- To validate your deployments api connection, see [Validating Your Deployment](../docs/validating-deployment.md)
+
+
+## Step 6: Configuring FiftyOne Enterprise Plugins
+
+FiftyOne Enterprise supports three plugin modes: **Builtin**, **Shared**, and **Dedicated**. Each offers different 
+trade-offs in isolation, flexibility, and resource management.
+
+### 🔹 1. Builtin Plugins Only (Default)
```

Comment:
Should we explain the benefit of builtin plugins (like we do for the other modes)?  Simplicity?